modules/humanStructure/main.cpp

```diff
@@ -42,8 +47,28 @@ class Processing : public yarp::os::BufferedPort<yarp::os::Bottle>
     yarp::os::RpcServer handlerPort;
     yarp::os::BufferedPort<yarp::sig::ImageOf<yarp::sig::PixelRgb> >    imageInPort;
     yarp::os::BufferedPort<yarp::sig::ImageOf<yarp::sig::PixelRgb> >    imageOutPort;
+    yarp::os::BufferedPort<yarp::sig::ImageOf<yarp::sig::PixelMono> >   imageOutDepthPort;
+    yarp::os::BufferedPort<yarp::sig::ImageOf<yarp::sig::PixelMono> >   imageOutSegmentPort;
+    yarp::os::BufferedPort<yarp::sig::ImageOf<yarp::sig::PixelFloat>>   imageInFloat;
     yarp::os::BufferedPort<yarp::os::Bottle >    targetPort;
+    yarp::os::BufferedPort<yarp::os::Bottle >    armPort;
```

Comment:
Maybe this `armPort` should be a `yarp::os::RpcClient`, as we typically send the `start` command to the rpc port of `googleSpeech`.

Reply:
Yes, this was what we agreed